src/webgl/p5.RendererGL.js

```diff
@@ -631,7 +635,22 @@ p5.RendererGL.prototype._getColorShader = function () {
   return this._defaultColorShader;
 };
 
+p5.RendererGL.prototype.getTexture = function (img) {
+  // efficiency?
+  var f = function(element) {
+    return element.src === img;
+  };
+
+  var tex = this.textures.find(f);
+  if (tex === undefined) {
+    tex = new p5.Texture(this, img);
+    this.textures.push(tex);
+  }
 
+  tex.set(img);
```

Comment:
To me, it makes sense to use `createTexture` or some analogue to it to do any texture creation or updating. That way, we don't concern ourselves with whether the data has changed when setting uniforms for rendering. The uniform setting just points to data that's already there.